### PR TITLE
Bug fixes & CustomBlock type for mob spawning

### PR DESCRIPTION
**Pull request by Baterka**
Fixes:
 - Everytime new phase was set, plugin sent the first block of the phase 2 times.
 - CustomBlocks in fixedBlocks were causing issues where it was pushing the CustomBlock multiple times into queue.
 - Phase config was not refreshed when doing /isa reload. I tested it and I do not see any issues.

Refactors:
 - Variable naming refactors, because having name `obPhase` for `phase` instance + name `phase` for the `ConfigurationSection` was not really understandable (I changed it to `phaseConfig`)

Added features:
 - New `CustomBlock` for mobs in format:
 - `type: mob mob: ZOMBIE underlying-block: GRASS_BLOCK`  It will spawn a mob on top of underlying block. I know that mob spawns are not exactly counted as "destroyed block", but in this case it is needed. It would not make sense setting it in fixedBlocks if not.
 - New `Requirement - cooldown` (seconds) that blocks new phase until time elapses.

### CODE REVIEW

File: src/main/java/world/bentobox/aoneblock/oneblocks/customblock/MobCustomBlock.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import world.bentobox.aoneblock.listeners.MakeSpace;
+import world.bentobox.aoneblock.oneblocks.OneBlockCustomBlock;
+import world.bentobox.bentobox.BentoBox;
+
+import java.util.*;
+
+/**
+ * A custom block that spawns mob on an underlying block
+ *
+ * @author HSGamer
```
Comment (HSGamer) [inline src/main/java/world/bentobox/aoneblock/oneblocks/customblock/MobCustomBlock.java:22]: You don't have to put my name on your code :rofl:

Comment (HSGamer) [inline src/main/java/world/bentobox/aoneblock/oneblocks/customblock/MobCustomBlock.java:22]:
```suggestion
 * @author Baterka
```

**Review by tastybento — APPROVED** [review]
This looks good and adds a nice cooldown feature. The units for the cooldown should be noted as they are seconds, but that's a minor issue.

**Comment by tastybento** [comment]
Thanks @Baterka !

**Pull request merged**